Documentation/devel/release.md

```diff
@@ -35,7 +35,7 @@ After merging and going back to master branch, we check out the release version
 
 - `git checkout HEAD^` should work; sanity check configure.ac (2nd line) after doing this
 - Build rkt, we'll use this in a minute:
-  - `git clean -ffdx && ./autogen.sh && ./configure --enable-tpm=no && make BUILDDIR=release-build -j 4`
+  - `git clean -ffdx && ./autogen.sh && ./configure --enable-tpm=no --with-report-file=configuration-report --with-stage1-default-images-directory=/usr/lib/rkt/stage1-images --with-stage1-default-location=/usr/lib/rkt/stage1-images/stage1-coreos.aci && make BUILDDIR=release-build -j4`
```

Comment:
So rkt will look in `/usr/lib/rkt/stage1-images` and fall back to the directory where `rkt` is located when looking for a stage1? (`/usr/bin`?)